source/core/shape/ovus.cpp

```diff
@@ -45,16 +45,18 @@
 *
 *  ovus
 *  {
-*     bottom_radius,top_radius
+*     bottom_radius,top_radius [distance d] [radius r] [precision p]   [sturm]
 *  }
 *
 *  The so long awaited 'Egg' forms.
 *
 *  Normally, the bottom_radius is bigger than the top_radius
-*  the center of the top sphere is at the zenith of the bottom sphere
+*  the center of the top sphere is at the zenith of the bottom sphere, unless
+*    a greater distance d is specified with "distance"
```

Comment:
Is the distance `d` an offset to the default position, or the Y(?) coordinate of the top sphere's center?